lms/djangoapps/homepage/helpers.py

```diff
@@ -1,25 +1,7 @@
-from django.contrib.auth.models import AnonymousUser
-
-from common.lib.nodebb_client.client import NodeBBClient
 from courseware.courses import get_courses
 from custom_settings.models import CustomSettings
-from edxmako.shortcuts import render_to_response
-from xmodule.modulestore.django import modulestore
-
 
-def home_page(request, user=AnonymousUser()):
-    """
-    Override the default homepage for students
-    """
-    featured_courses = get_featured_courses(user)
-    status_code, featured_categories = NodeBBClient().categories.featured()
-
-    context = {
-        'courses': featured_courses,
-        'categories': featured_categories if status_code == 200 else [],
-    }
-
-    return render_to_response("homepage/homepage.html", context)
+from xmodule.modulestore.django import modulestore
 
 
 def get_featured_courses(user):
```

Comment:
lets check if this method is not being used anywhere else. If it is not, lets remove it!

Reply:
Also lets remove featured categories from nodebb side and edx Nodebb client

Reply:
@saad-waqar actually @emraan004 suggested to move it to helpers for future uses and due to same reason I didn't remove category method.

Reply:
Understood! Lets keep this code in case it is required in future.